### ISSUE TITLE
Incomplete stories cannot be rescheduled for another iteration

### ISSUE THREAD

**Issue by pacso** [SphereSoftware/easy_agile#18]
Currently stories cannot be moved to another iteration. If a story is not completed during a planned iteration it must be recreated from scratch.

It would be easier if you could simply add incomplete stories to new iterations if required.

**Comment by mjones-rpx** [comment]
I agree with this. We can't use this right now, because as far as I can tell you can't add a story from the backlog to an iteration that's already active, either, which doesn't make sense to me. If velocity is higher than expected, stories need to flow from the backlog into the iteration, but it looks like this isn't possible?
